lib/game.rb

```diff
@@ -0,0 +1,14 @@
+class Game
+  attr_reader :options
+
+  def initialize
+    @options = []
+    @options << 'rock'
+    @options << 'paper'
+    @options << 'scissors'
+  end
```

Comment:
Kate, I was just wondering why you decided to not do @options = ['rock', 'paper', 'scissors']?